account_invoice_import/wizard/account_invoice_import_error_view.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp><data>
+
+    <record model="ir.ui.view" id="import_no_vat">
+        <field name="name">account.invoice.import.no_vat.view</field>
+        <field name="model">account.invoice.import.error</field>
+        <field name="type">form</field>
+        <field name="arch" type="xml">
+            <form string="No VAT matched">
+                <group>
+                    <p>There is no matching vat found, please set this by yourself.</p>
```

Comment:
This looks so passive aggressive :smile: `There is no matching VAT found, please configure this on the invoice manually.`

Reply:
We need to be hard for business 🤣